src/api/handlers/user.ts

```diff
@@ -0,0 +1,14 @@
+import { RequestManager } from "../../rest/request_manager.ts";
+import { UserPayload } from "../../types/mod.ts";
+import { endpoints } from "../../util/constants.ts";
+
+/**
+ * Returns the user object of the requester's account.
```

Comment:
what happens when this function is called? what values are u getting back?

Reply:
What Skillz meant is the you should cast that to the return type...